### DISCUSSION TITLE
请问如何添加Google Play的快捷方式到桌面（开始菜单）How can I add a shortcut of GooglePlay to the desktop or startmenu

### DISCUSSION THREAD

**Discussion by KimmyLueng** [LSPosed/MagiskOnWSALocal#241] [Q&A] [answered]
如题，每次进入Google Play都要通过其他方法进去

**Comment by Howard20181** [comment, marked as answer]
WSA里面安装的所有应用都创建了快捷方式到开始菜单的所有应用页面，你可以右击选择固定到“开始”屏幕。
![image](https://user-images.githubusercontent.com/40033067/199688658-535b655f-a910-459c-9a33-ba1610eb8f03.png)
![image](https://user-images.githubusercontent.com/40033067/199688863-753127f4-0f23-4bd6-aac6-b9cc7be0f990.png)

**Comment by KimmyLueng** [comment]
![image](https://user-images.githubusercontent.com/54270610/199689458-9bb06cde-2f24-4dd9-9981-2ef94790afb8.png)
我的没有，有办法找回吗？

**Reply by KimmyLueng** [reply]
![image](https://user-images.githubusercontent.com/54270610/199689765-d2bd7caa-3fad-4dc9-b136-46d306005619.png)

**Reply by Howard20181** [reply]
你是怎样安装WSA的？是不是本来就没有Play商店

**Reply by KimmyLueng** [reply]
就按你的方法安装，我记得第一次安装完是有play商店快捷方式，但后面重启后就没了

**Reply by Doddddd** [reply]
https://github.com/LSPosed/WSAHelper

**Reply by Howard20181** [reply]
现在WSAHelper不工作

**Reply by Doddddd** [reply]
创建快捷方式，填上 %localappdata%\Microsoft\WindowsApps\MicrosoftCorporationII.WindowsSubsystemForAndroid_8wekyb3d8bbwe\WsaClient.exe /launch wsa://com.android.vending（谷歌商店的包名）

改图标：右键属性，更改图标，填上%localappdata%\Packages\MicrosoftCorporationII.WindowsSubsystemForAndroid_8wekyb3d8bbwe\LocalState\com.android.vending.ico，确定，确定，应用，确定
